landingzones/tests/test_permissions.py

```diff
@@ -200,6 +200,83 @@ def test_zone_create_disable(self):
         self.assert_response(url, good_users, 200)
         self.assert_response(url, bad_users, 302)
 
+    def test_zone_update(self):
```

Comment:
Make sure to add new users introduced in SODAR Core v0.13.0 to all permission tests:

```
self.user_delegate_cat,
self.user_contributor_cat,
self.user_guest_cat,
self.user_finder_cat
```

The `user_finder_cat` role should generally no permissions in any project app views so it should basically always go under `bad_users`.